src/Controls/src/Core/TitleBar/TitleBar.cs

```diff
@@ -0,0 +1,301 @@
+﻿using Microsoft.Maui.Graphics;
+
+namespace Microsoft.Maui.Controls
+{
+	public partial class TitleBar : ContentView, ITitleBar
```

Comment:
Review inheriting from View or Element vs ContentView?

Reply:
Or at the very least TemplatedView to avoid the overriding of Content.

Reply:
Updated w/ TemplatedView